utils/metta.py

```diff
@@ -0,0 +1,69 @@
+from datetime import date
+from calendar import monthrange
+import metta
+import pandas as pd
+from random import randint
+
+def quarter_boundaries(quarter):
```

Comment:
Let's make a point to talk about the metta integration soon. I know this was a stopgap solution. We should be fine with merging the stopgap solution, but I think we should figure out what exactly we want to do next with this.